projects/www/src/app/examples/store-walkthrough/src/book-collection/book-collection.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Book } from '../book-list/books.model';
+
+@Component({
+  selector: 'app-book-collection',
+  template: `
+    @for(book of books; track book) {
```

Comment:
All templates are inline to run the example in vite/stackblitz.
This also means that no css is imported.